README.md

```diff
@@ -1,13 +1,19 @@
 # Happy Thoughts
 
-Replace this readme with your own information about your project.
+This week we created a Twitter-ish app that fetches from a public API with three endpoints to show the last 20 happy-thought messages, new messages and the number of likes on the individual messages.  
+We learned to use useEffect to do the fetch of the old happy-thought messages at the exakt time when the fetch-component was mounted. 
+We were given a design to follow for our app which consists of a form to post new happy-thoughts (6-140 characters or disabled button), previous happy-thoughts messages with a heart button rendering likes when pushed and that changes color when at least one like. 
+It also shows how many likes the message has recieved and how long ago it was posted.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Regarding accessability I did cross-browser testing on Crome, Firefox and Edge and tabbing/enter works with typing, posting and liking on all of them. I used semantic html when possible and aria span and labels on the emojis. 
+I used https://validator.w3.org/nu/ for validating, and https://wave.webaim.org/ helped me add a htmlFor label on the headline of my form to be better for screenreaders.
```

Comment:
Really nice that you made these validations and justified accordingly  - it made me use the wave.webaim.org today and realized that I had to do some additional labelling - thanks!